core/new-gui/src/app/workspace/component/navigation/navigation.component.scss

```diff
@@ -140,6 +140,35 @@
       }
     }
 
+    .execution-name {
```

Comment:
can you add comments to explain what these styles are for?

Reply:
done